src/adapters/catalyst-client.ts

```diff
@@ -59,5 +60,24 @@ export async function createCatalystClient({
     return retry(executeClientRequest, retries, waitTime)
   }
 
-  return { getProfiles, getProfile }
+  async function getOwnedNames(
+    address: EthAddress,
+    options: ICatalystClientRequestOptions = {}
+  ): Promise<OwnedNames[]> {
+    const { retries = 3, waitTime = 300, lambdasServerUrl } = options
+    const executeClientRequest = rotateLambdasServerClient(
+      (lambdasClientToUse) => lambdasClientToUse.getNames(address),
```

Comment:
Should we use a small page size here since we only want to know if the user owns at least one name?

Maybe we can accept a new parameter of type `GetNamesParams` in the `getOwnedNames`, so the logic component can use a page size of one. WDYT?

Reply:
will do, great catch